src/final2/subtests/InvalidCommandTest.java

```diff
@@ -0,0 +1,915 @@
+package final2.subtests;
+
+import org.junit.Test;
+
+import test.Input;
+import test.runs.ErrorRun;
+import test.runs.NoOutputRun;
+import test.runs.Run;
+
+/**
+ * Tests the program for error messages with not well-formed or invalid commands.
+ * 
+ * @author Roman Langrehr
+ * @since 26.03.2015
+ * @version 1.0

```

Comment:
We had previously come to the conclusion `@version` tags were not used in a meaningful way in our tests.